netaddr.go

```diff
@@ -1132,6 +1132,39 @@ func (s *IPSet) AddRange(r IPRange) {
 	s.in = append(s.in, r)
 }
 
+// RemoveFreePrefix removes and returns a Prefix of length bits from the IPSet.
+// The Prefix is also removed from the IPSet.
```

Comment:
this sentence seems redundant with the two "remove" words on the previous line (which, while redundant, are conventional)

Reply:
done and rebased